### PR TITLE
bug in run benchmarks

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Goal
The goal of this PR is to fix a bug in benchmark script

Build binary while running benchmarks with all features

### CODE REVIEW

File: scripts/run_all_benchmarks.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ function run_benchmark() {
   --template=$3
 }
 
-cargo build --profile production --features runtime-benchmarks --features frequency --workspace || exit_err
+cargo build --profile production --features runtime-benchmarks --features all-frequency-features --workspace || exit_err
```
Comment (dopatraman) [inline scripts/run_all_benchmarks.sh:29]: why dont we add this line to the Makefile? that way we reference it one place for other build scripts

Comment (saraswatpuneet) [reply scripts/run_all_benchmarks.sh:29]: I need to check with DevOps if they are calling this script as is else they will have to update Jenkins job for the same too

**Comment by dopatraman** [comment]
I see this error when running the msa benchmarks:
```
$ ./target/release/frequency benchmark pallet \
  --pallet pallet_msa \
  --extrinsic "*" \
  --chain="frequency" \
  --execution wasm \
  --wasm-execution compiled \
  --steps 50 \
  --repeat 10 \
  --output=./output \
  --template=./.maintain/frame-weight-template.hbs

Error: Input("Benchmarking wasn't enabled when building the node. You can enable it with `--features runtime-benchmarks`.")
```

Note: This is after running
```
cargo build --profile production --features runtime-benchmarks --features all-frequency-features --workspace
```

**Comment by saraswatpuneet** [comment]
@dopatraman  you got it wrong 

The binary is not in release folder but in production folder
So it is ./target/production/frequency

**Comment by saraswatpuneet** [comment]
Ok this PR has an addition, updated script to run for a specified package

File: scripts/run_all_benchmarks.sh (hunk 1 above)
Comment (wilwade) [reply scripts/run_all_benchmarks.sh:28]: @saraswatpuneet It uses `make benchmarks`

Comment (saraswatpuneet) [reply scripts/run_all_benchmarks.sh:28]: @wilwade  yes, question should be build all features for benchmarks or just frequency like it was before this PR

**Pull request merged**